Location input and validation

aru456 opened the pull request.

maludee commented:
@aru456 can you add a description to the PR & describe whats changed so it's easy to tell what new stuff to test/check? 
based on the commits I figured this PR is for long/lat in the landlord submission form as well as the max/min price bug, so that's what I tested. 
- I tried to test out the location validation in the landlord submission form but it's using my browser's location, not based on the apartment building address I inputted as landlord. I think there would be cases where the landlord would not be in his own building at the time of posting, is it possible to make it check the address he inputs instead? 
- Max and min price validation looks good

aru456 commented:
@maludee I've added the location input with GOOGLE Places Api, and also stripped the address till the first house number and street, and added the #apt_no to the address. 

Modified the way we are taking the location, and also taking the input of the apt number from user since there is no column for the Apt#, I've concatenated with the google address itself (but the Borough, NY,USA) part is ignored in the address as we are already filling the

maludee reviewed (approved):
looks good to me! one thing I noticed - if I uploaded a weird sized image, it kind of messed up our listing cards. see this apartment as an example: 111 West 56th Street #41A

I think we should merge this and fix that as a separate ticket though, because it's just a minor aesthetic issue
